onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -986,6 +999,12 @@ def _where_input_wrangler(
         matcher=lambda sample: len(sample.input.shape) == 0,
         reason="fixme: Rank(0) input will lead ORT failed due to different rank(result) in if-else branch",
     ),
+    skip(
+        "scatter_reduce",
+        matcher=lambda sample: sample.kwargs.get("include_self") is False,  # ORT only support include_self=True mode
+        # or len(sample.input.shape) == 0,
+        reason="ORT does't support include_self=False option, and skip rank 0 case because failed in FullGraph mode",
```

Comment:
```suggestion
        reason="ORT does't support include_self=False option",
```

looks like len(sample.input.shape) == 0 was commented out?